src/components/NavbarDesktop.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <el-header id="navbar">
+        <img id="img-navbar"  src="../assets/openpromises.png" height="48px"/>
```

Comment:
since this already has an id, for future cases, please set the height in the <style> section below